README.md

```diff
@@ -34,146 +34,123 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
-
-2. Mount the block volume and format it
-
-    ```bash
-    sudo ./rook block mount --name test --path /tmp/rook-volume
-    sudo chown $USER:$USER /tmp/rook-volume
-    ```
-
-3. Write and read a file
-
-    ```bash
-    echo "Hello Rook!" > /tmp/rook-volume/hello
-    cat /tmp/rook-volume/hello
-    ```
+### Kubernetes
 
-4. Cleanup
+This example shows how to build a simple, multi-tier web application on Kubernetes using persistent volumes enabled by Rook.
 
-    ```bash
-    sudo ./rook block unmount --path /tmp/rook-volume
-    ```
+#### Prerequisites
 
-#### Shared File System
-1. Create a shared file system
+This example requires a running Kubernetes cluster.
 
-    ```bash
-    ./rook filesystem create --name testFS
-    ```
+You will also need to modify the kubelet service to bind mount `/sbin/modprobe` to allow access to `modprobe`. Access to modprobe is necessary for using the rbd volume plugin (<https://github.com/kubernetes/kubernetes/issues/23924>).
+If using RKT, you can allow modprobe by following this [doc](https://github.com/coreos/coreos-kubernetes/blob/master/Documentation/kubelet-wrapper.md#allow-pods-to-use-rbd-volumes).  
 
-2. Verify the shared file system was created
+For a quick start, use <https://github.com/rook/coreos-kubernetes/tree/rook-demo> (rook-demo branch in the rook fork of the coreos-kubernetes repo). This will bring up a multi-node Kubernetes cluster, configured for using the rbd volume plugin.
 
-   ```bash
-   ./rook filesystem ls
-   ```
+```
+$ git clone https://github.com/rook/coreos-kubernetes.git
+$ cd coreos-kubernetes
+$ git checkout rook-demo
+```
 
-3. Mount the shared file system from the cluster to your local machine
+You can now get a vagrant kubernetes cluster running like this (from the root of the repo):
 
-   ```bash
-   ./rook filesystem mount --name testFS --path /tmp/rookFS
-   sudo chown $USER:$USER /tmp/rookFS
-   ```
+```
+$ cd multi-node/vagrant
+$ vagrant up
+$ export KUBECONFIG="$(pwd)/kubeconfig"
+$ kubectl config use-context vagrant-multi
+```
 
-4. Write and read a file to the shared file system
+Then wait for the cluster to come up and verify that kubernetes is done initializing (be patient, it takes a bit):
 
-   ```bash
-   echo "Hello Rook!" > /tmp/rookFS/hello
-   cat /tmp/rookFS/hello
-   ```
+```
+$ kubectl cluster-info
+```
 
-5. Unmount the shared file system (this does **not** delete the data from the cluster)
+If you see a url response, you are ready to go.
 
-   ```bash
-   ./rook filesystem unmount --path /tmp/rookFS
-   ```
+#### Deploy Rook
 
-6. Cleanup the shared file system from the cluster (this **does** delete the data from the cluster)
+Rook can be setup and deployed in Kubernetes by simply deploying the [rook-operator](https://github.com/rook/rook/blob/master/demo/kubernetes/rook-operator.yaml) deployment manifest.
 
-   ```
-   ./rook filesystem delete --name testFS
-   ```
+```
+$ kubectl create namespace rook
+$ kubectl create -f rook-operator.yaml
```

Comment:
i prefer having instructions that the user can just copy/paste.  there should be a `cd` command here or we should use the more complete path to the rook-operator yaml file `./demo/kubernetes/rook-operator.yaml'.  as it is here, i think the user would get an error about `rook-operator.yaml` not found.